conmon/conmon.c

```diff
@@ -1085,6 +1089,30 @@ static void setup_oom_handling(int container_pid)
 	g_unix_fd_add (oom_event_fd, G_IO_IN, oom_cb, NULL);
 }
 
+static void do_exit_command()
+{
+	gchar **args;
+	size_t n_args = 0;
+
+	/* Count the additional args, if any.  */
+	if (opt_exit_args)
+		for (; opt_exit_args[n_args]; n_args++);
+
+	args = g_malloc (sizeof (gchar *) * (n_args + 2));
+
+	args[0] = opt_exit_command;
+	if (opt_exit_args)
+		for (n_args = 0; opt_exit_args[n_args]; n_args++)
```

Comment:
style nit, not a big fan of reusing n_args here, I'd pref a separate int.

Reply:
why would that be a problem?  The `n_args` variable is used to iterate the same array

Reply:
For me I read into the for quickly, missed that you were reusing the n_args in the for and was wondering why you were reversing the args.  Again it's not wrong as is, I just find it clearer to have a separate int for an iterator, your mileage may vary.